src/chapters/orm/studio.rst

```diff
@@ -0,0 +1,114 @@
+Studio: Creating an ``AbstractEntity``
+======================================
+
+In :ref:`orm1-exercises`, you created a new persistent class, ``EventCategory``. We will eventually use this class to categorize events in our application. However, your application now has a fair amount of duplication across the two model classes. This studio has you reduce repetition (i.e. "DRY out your code") using inheritance and abstract classes.
+
+Getting Ready
+-------------
+
+In your ``coding-events`` repository, checkout out the `add-persistent-category branch <https://github.com/LaunchCodeEducation/coding-events/tree/add-persistent-category>`_ and create a new branch, ``abstract-entity-studio``, off of that branch.
```

Comment:
Considering this a second time reminds me of the Angular issue where students forked and cloned the same repository several times.  This occurred when they followed the link to the branch in the repository instead of checking out that same branch locally.

I'd recommend removing the link and reminding the students that they already have access to the branch (so long as they pulled down any changes).